src/data/navigation.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "label": "Invest",
+    "url": "#"
+  },
+  {
+    "label": "Insure",
+    "url": "#"
+  },
+  {
+    "label": "Goals",
+    "url": "#"
+  },
+  {
+    "label": "Partners",
+    "url": "#"
+  }
+]
```

Comment:
Please add an empty new line at the end of the page.

Reply:
I added a new empty line at the end of the file.